Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -35,9 +35,8 @@ void Jit64::psq_stXX(UGeckoInstruction inst)
   int w = indexed ? inst.Wx : inst.W;
   FALLBACK_IF(!a);
 
-  auto it = js.constantGqr.find(i);
-  bool gqrIsConstant = it != js.constantGqr.end();
-  u32 gqrValue = gqrIsConstant ? it->second & 0xffff : 0;
+  bool gqrIsConstant = js.constantGqrValid[i];
+  u32 gqrValue = gqrIsConstant ? (js.constantGqr[i] & 0xffff) : 0;
```

Comment:
You can skip the ternary if you don't define `gqrValue` until after `gqrIsConstant` has been checked.